tests/prague/eip7702_set_code_tx/spec.py

```diff
@@ -28,7 +28,7 @@ class Spec:
 
     SET_CODE_TX_TYPE = 0x04
     MAGIC = 0x05
-    PER_AUTH_BASE_COST = 2_500
+    PER_AUTH_BASE_COST = 15_000
```

Comment:
```suggestion
    PER_AUTH_BASE_COST = 12_500
```

https://github.com/ethereum/EIPs/pull/8929 changed it to 15k, but the follow up changed it to 12.5k:

https://github.com/ethereum/EIPs/pull/8948/files#diff-2e55aa3f36cb003b2064245d31124e5b9c0ce17853f0606ae2a820f0effdfd83R34

I had two single-signer-single-auth tests passsing but "test_gas_cost[fork_Prague-blockchain_test-multiple_valid_authorizations_single_signer]" failed, if I change this then it passes.

The single signer single auth passes because it authorizes an empty account. The multi-auth single-signer thus first creates an account and then it will update it (thus getting the refund).